packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -660,3 +662,44 @@ You then need to associate the registered Inline Hook with a Custom Authorizatio
 
 > **Note:** Only one Inline Hook can be associated with each rule.
 
+## Troubleshooting
+
+This section covers what happens when a token inline hook flow fails either due to the external inline hook service returning an error object or not returning a successful response, or the inline hook patch fails.
+
+### When there is an error
```

Comment:
Nit: it is better that the errors are listed in following order:
- communication error;
- http response code is not 200;
- error object in response;
- a command cannot be performed